ex5/server/db/migrations/20220618070848-create-items.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = {
+	async up(queryInterface, Sequelize) {
+		return queryInterface.addColumn('vdo', 'avatar', {
```

Comment:
this line actually crashes the db:migration , not sure what purpose it has - the project ran successfully after I've commented it out